templates/kube-scheduler.yaml

```diff
@@ -32,6 +32,35 @@ spec:
     volumeMounts:
     - name: kube-scheduler-policy-config
       mountPath: {{kube_scheduler_policy_config_dir}}
+  {% if ext_scheduler_image and ext_scheduler_label_key and ext_scheduler_log_level and ext_scheduler_log_insight_host -%}
+  - name: extended-scheduler
+    image: {{ext_scheduler_image}}
+    env:
+    - name: LABEL_KEY
+      value: {{ext_scheduler_label_key}}
+    - name: LOG_LEVEL
+      value: {{ext_scheduler_log_level}}
+    - name: LOG_INSIGHT_HOST
+      value: {{ext_scheduler_log_insight_host}}
+    livenessProbe:
+      httpGet:
+        host: 127.0.0.1
+        path: /v1.0/liveliness
+        port: 9090
+      initialDelaySeconds: 20
+      periodSeconds: 30
+      timeoutSeconds: 5
+    ports:
+    - name: http
+      containerPort: 9090
+    resources:
+      limits:
+        memory: {{ext_scheduler_limits_memory}}
+        cpu: {{ext_scheduler_limits_cpu}}
+      requests:
+        memory: {{ext_scheduler_requests_memory}}
+        cpu: {{ext_scheduler_requests_cpu}}
+  {% endif -%}
   volumes:
```

Comment:
The below volume should be created only if 'kube_scheduler_policy_config_dir' is set